crates/typst/src/eval/dict.rs

```diff
@@ -167,6 +167,20 @@ impl Dict {
         Arc::make_mut(&mut self.0).insert(key, value);
     }
 
+    /// Inserts a new pair into the dictionary and return the value. If the
```

Comment:
The description is a bit strange. The first sentence sounds like it unconditionally inserts. It should start with the fact that it retrieves the value and then go on an explain what happens if the value wasn't present.